index.rst

```diff
@@ -6,24 +6,474 @@
+or an explanation of why the request was not implemented. 
+
+Contact the reviewers to let them know the pull request now awaits their approval.
+
+Merge
+-----
+
+After the reviewers have approved the pull request, rebase and merge your ticket branch into the ``main`` branch (*not* to prod).
+Resolve all conlicts, if there are any.
+After the successful merge, delete your branch.
+
+Release to ``prod``
+-------------------
+
+To "release" the new version of ``main`` to ``prod`` branch (i.e., to update all RSP users' tutorial notebooks),
+delete the current ``prod-prior-to-rebranch`` branch, rename ``prod`` as ``prod-prior-to-rebranch``, then create a new ``prod`` branch from ``main``.
+Doing it this way avoids weird git issues that cause conflicts in ``main`` to ``prod`` merges.
+
+The number of pushes to the ``prod`` branch should be minimized.
+E.g., if there are a few tickets being completed within a week, coordinate with other notebook developers to collect all changes in
+the ``main`` branch, and then do a single release to ``prod``.
+
+Jira tickets
+------------
+
+Remember to make comments in the associated Jira tickets about the major updates and mark the ticket as done.
+
+
+Updates to the RSP's recommended version
+----------------------------------------
+
+Decisions on whether to bump the recommended image for the RSP are made jointly between the CET and the RSP team.
+Once the decision has been made, a PREOPS ticket will be created and assigned to a CET member.
+
+Bumping the recommended always occurs during a Patch Thursday window.
+The notebook updates should be merged to the ``main`` branch by the day before.
+
+The workflow is to create a new branch of the ``tutorial-notebooks`` repository from the ``main`` branch,
+test all of the notebooks with the new version, and make updates as needed.
+
+Do not suppress warnings while testing.
+It is not necessary to use the ``flake8`` 'magic' commands while testing, unless significant changes to the code are required.
+
+At minimum, the header will have to be updated with a new date and verified version.
+Ensure that all notebooks are cleared before committing new versions.
+
+When the updates are complete, use a new pull request to merge the branch into ``main``. 
+A review is not typically needed at this stage.
+
+During the Patch Thursday window, after the recommended has been bumped, release to ``prod`` following the steps above.
+
+Remember to set the PREOPS ticket to done.
+
+
+Portal tutorial format and style
+================================
+
+The portal tutorials are written in RST format and are kept within the data release documentation at
+`dp0-2.lsst.io/tutorials-examples <https://dp0-2.lsst.io/tutorials-examples/index.html>`__.
+
+All portal tutorials should have a descriptive title, and list the contact authors, the date last verified to run, and the targeted learning level.
```

Comment:
And the RSP/code version that was last used to run?

Reply:
added